.github/workflows/system-tests.yml

```
@@ -7,7 +7,6 @@ on:
     - cron:  '00 04 * * 2-6'
   push:
     branches:
-      - master
       - 0.x
```

Are we planning to support auto injection in 0.x?
```suggestion
      - 0.x
```

---

No, 0.x is only supported for bug fixes - no new features